content/en/blog/2025/ai-agent-observability/index.md

```diff
@@ -0,0 +1,258 @@
+  etc.
+- Option 2: External instrumentation in OpenTelemetry owned repository, like
+  [instrumentation-genai](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation-genai)
+  etc.
+
+Both options work well, but the long term goal is to host the code in
+OpenTelemetry owned repositories, like Traceloop is trying to
+[donate the instrumentation code](https://github.com/open-telemetry/community/issues/2571)
+to OpenTelemetry now.
+
+As a developer of an agent framework, here are some pros and cons of
+instrumentation with OpenTelemetry:
+
+- Pros
+  - Decouples observability from the core framework, reducing bloat.
+  - Leverages OpenTelemetry’s community-driven maintenance for instrumentation
+    updates.
+  - Allows users to mix and match contrib libraries for their specific needs
+    (e.g., cloud providers, LLM vendors).
+  - More likely to leverage best practices around semantic conventions and
+    zero-code instrumentation
+- Cons
+  - Risk of fragmentation if users rely on incompatible or outdated contrib
+    packages for both install time and runtime.
+  - Development velocity slows down when there are too many PRs in the
+    OpenTelemetry review queue.
+- Best practices for this approach:
+  - Ensure compatibility with popular OpenTelemetry contrib libraries (e.g., LLM
+    vendors, vector DBs).
+  - Provide clear documentation on recommended contrib packages and
+    configuration examples.
+  - Avoid reinventing the wheel; align with existing OpenTelemetry standards.
+- As a developer of an agent application, you may want to choose an agent
+  framework with baked-in instrumentation if…
+  - You need fine-grained control over telemetry sources and destinations.
+  - Your use case requires integrating observability with niche or custom tools.
+
+**NOTE:** Regardless of the approach taken, it is essential that all AI agent
+frameworks adopt the AI agent framework semantic convention to ensure
+interoperability and consistency in observability data.
+
+## The future of AI agent observability
+
+Looking ahead, AI agent observability will continue to evolve with:
+
+- **More robust semantic conventions** to cover edge cases and emerging AI agent
+  frameworks.
+- **A unified AI agent framework semantic convention** to ensure
+  interoperability across different frameworks while allowing flexibility for
+  vendor-specific extensions.
+- **Continuous improvements to the AI agent semantic convention** to refine the
+  initial standard and address new challenges as AI agents evolve.
+- **Improved tooling** for monitoring, debugging, and optimizing AI agents.
+- **Tighter integration with AI model observability** to provide end-to-end
+  visibility into AI powered applications.
+
+## The role of OpenTelemetry's GenAI SIG
+
+The **GenAI Special Interest Group (SIG) in OpenTelemetry** is actively defining
+a set of **GenAI semantic conventions**, which encompasses several key areas:
```

Comment:
How about linking to the existing semconv?

```suggestion
[GenAI semantic conventions](/docs/specs/semconv/gen-ai/) that cover key areas such as:
```

Reply:
Done